### PR TITLE
Format .yml and .md with prettier

### PR DESCRIPTION
**Pull request by gavinelder**
From discussion on https://github.com/panther-labs/panther/pull/61 about linters. 

[GoFumpt](https://github.com/mvdan/gofumpt) & `gofumports`
[prettier](https://prettier.io/)

```shell
gofumpt -w .
prettier --write "**/*.md"
prettier --write "**/*.yml"
```

Seems the CI failures are historical.

### CODE REVIEW

**Comment by gavinelder** [comment]
@austinbyers  Np feel free to close when you feel it's ready I have no expectation of this ever landing just wanted to share a quick demo.

**Comment by austinbyers** [comment]
For sure! Thanks for showing! I'd definitely like to use one or both tools.

I was just providing more context for you and the team about why CI is failing and what needs to happen to actually use these

File: internal/compliance/snapshot_scheduler/scheduler/schedule_test.go [hunk 1]
```diff
@@ -89,86 +89,15 @@ func getTestInvokeOutput(payload interface{}, statusCode int64) *lambda.InvokeOu
...
+	{SourceIntegrationMetadata: &models.SourceIntegrationMetadata{IntegrationID: aws.String("2654cf7a-a13a-4b9b-8b4d-f3e5bfc51cb4"), IntegrationLabel: aws.String("Development AWS Account"), IntegrationType: aws.String("aws-scan"), ScanIntervalMins: aws.Int(60), ScanEnabled: aws.Bool(true)}, SourceIntegrationStatus: &models.SourceIntegrationStatus{ScanStatus: aws.String("scanning")}, SourceIntegrationScanInformation: &models.SourceIntegrationScanInformation{
+		LastScanStartTime: aws.Time(time.Now().Add(time.Duration(-65) * time.Minute)), // Last time it scanned was a day ago.
+		LastScanEndTime:   aws.Time(time.Now().Add(time.Duration(-24) * time.Hour)),
+	}},
+}
```
Comment (austinbyers) [inline internal/compliance/snapshot_scheduler/scheduler/schedule_test.go]: 👎 👎 👎

File: internal/core/alert_delivery/delivery/dispatch_test.go [hunk 2]
```diff
@@ -184,7 +184,7 @@ func TestDispatchUseNonCachedDefault(t *testing.T) {
 
 	mockLambdaClient.On("Invoke", mock.Anything).Return(mockLambdaResponse, nil)
 	alert := sampleAlert()
-	alert.OutputIDs = nil //Setting OutputIds in the alert to nil, in order to fetch default outputs
+	alert.OutputIDs = nil // Setting OutputIds in the alert to nil, in order to fetch default outputs
```
Comment (austinbyers) [inline internal/core/alert_delivery/delivery/dispatch_test.go]: 👍

File: internal/compliance/alert_processor/models/api.go [hunk 3]
```diff
@@ -25,18 +25,18 @@ import (
 // ComplianceNotification represents the event sent to the AlertProcessor by the compliance engine.
 type ComplianceNotification struct {
 
-	//ResourceID is the ID specific to the resource
+	// ResourceID is the ID specific to the resource
```
[the remark this answers is not in the document]
Comment (austinbyers) [reply internal/compliance/alert_processor/models/api.go]: Agreed, it's a nice check we don't have elsewhere, but sadly `gofumpt` doesn't let you selectively enable its decisions

**Comment by gavinelder** [comment]
@austinbyers YML has been linted and force pushed.

File: deployments/compliance/remediation.yml [hunk 4]
```diff
@@ -228,30 +226,29 @@ Resources:
       Runtime: go1.x
       Timeout: !Ref ProcessorTimeoutSec
       Policies:
-        - !If [TracingEnabled, 'arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess', !Ref 'AWS::NoValue']
-        -
-          Id: ReceiveAndDeleteSQSMessages
+        - !If [
+            TracingEnabled,
+            'arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess',
```
[the remark this answers is not in the document]
Comment (austinbyers) [reply deployments/compliance/remediation.yml]: Synced with @nhakmiller , we'll merge this first

**Pull request merged**